### PR TITLE
Add Request a Priority Review page

### PR DESCRIPTION
**Pull request by cvalarida**
image

Some outstanding questions:
- Should we use `h3`s?
  - @1Copenut advises us to go in order, h1 -> h2 -> h3, etc. for accessibility reasons, but it looks kinda gross
  - Also, it looks like a lot of other static content pages skip heading levels, so while this isn't great like it is now, it _is_ consistent
- Should the "By mail" and "By fax" be headers or just bold?
  - I'm leaning toward just bold
- Is the space between the "By mail" and "By fax" headers and content okay?

@bethpotts Gina said you wanted to review the content again; feel free to add any comments here if you're ready with them.

### CODE REVIEW

**Comment by gnakm** [comment]
Could we wrap the blue box to only include from "One of these must be true..." and the end of "Notes:...."

**Comment by cvalarida** [comment]
@gnakm Done!
image

**Comment by bethpotts** [comment]
@cvalarida @gnakm 
1) To answer one of the above questions, we use #### for the subhead for "By mail" and "By fax"

2) Gina, your request to make the blue box smaller actually makes it less consistent with all our other static pages, so I'd recommend reverting that. (We always have a question at the top of the blue box.)

3) Here's the code for the address block:
<p class="va-address-block">
Board of Veterans’ Appeals<br>
PO Box 27063<br>
Washington, DC 20038<br>
</p>

**Comment by cehsu** [comment]
Good question about the accessibility consideration with the heading order, would it make sense to file a ticket to style those so that we can use them in the appropriate order?

File: content/pages/disability-benefits/claims-appeal/request-a-priority-review.md
Comment (cehsu) [inline content/pages/disability-benefits/claims-appeal/request-a-priority-review.md]: Is there an extra space before `=`?

File: content/pages/disability-benefits/claims-appeal/request-a-priority-review.md
Comment (cehsu) [inline content/pages/disability-benefits/claims-appeal/request-a-priority-review.md]: Should this be "75 years old or older"?

File: content/pages/disability-benefits/claims-appeal/request-a-priority-review.md
Comment (cehsu) [inline content/pages/disability-benefits/claims-appeal/request-a-priority-review.md]: We occasionally include the country code in fax numbers (e.g. `pension-management-center.md`), should we do so here?

**Review by cehsu — APPROVED** [review]
LGTM once @gnakm @bethpotts approve.

File: content/pages/disability-benefits/claims-appeal/request-a-priority-review.md
Comment (cvalarida) [reply content/pages/disability-benefits/claims-appeal/request-a-priority-review.md]: Copy pasta. Good eye. :female_detective:

**Comment by cvalarida** [comment]
@bethpotts I think the trouble I'm having with putting "By mail" and "By fax" in a header is the parent section isn't in a header ("You or your representative can send your written request in 1 of 2 ways:"), so this makes kind of a weird hierarchy both visually and for screen readers.

### How do I request to have my appeal Advanced on Docket?
h3
#### You’ll need to include all of this information
h4 instead of bold
#### You or your representative can send your written request in 1 of 2 ways
h4 instead of bold
##### By mail
h5
##### By fax
h5

**Comment by gnakm** [comment]
@cvalarida The mail and fax should be styled like this: 
image

https://www.vets.gov/health-care/apply/

**Comment by cvalarida** [comment]
@gnakm Heh, I was reviewing one of Claire's PRs and saw that formatting was a thing. One of these days, I'd love to make a list of all the styling we have.

I can have dreams, right? :stuck_out_tongue_winking_eye: 

After a few changes, this is what I've got:
image

The changes are as follows:
- I added the `h4`s as I described above
  - If we want to change that, that's fine by me; I just thought I'd show what I was talking about
  - I do think this both looks better and makes more sense for header navigation in screen readers
- The mailing address is formatted properly now
- Removed the `:` after `Fax to`
- Removed `Send to:`

File: content/pages/disability-benefits/claims-appeal/request-a-priority-review.md
Comment (gnakm) [reply content/pages/disability-benefits/claims-appeal/request-a-priority-review.md]: yes

File: content/pages/disability-benefits/claims-appeal/request-a-priority-review.md
Comment (cvalarida) [inline content/pages/disability-benefits/claims-appeal/request-a-priority-review.md]: :thinking: So is it >= 75, or just == 75?

Previously, we say "75 or older", and here it's just "older than 75."

Comment (gnakm) [reply content/pages/disability-benefits/claims-appeal/request-a-priority-review.md]: Could you change this to "75 or older"

**Comment by cehsu** [comment]
Adding @bethpotts as a reviewer as she's made several recommendations.

**Review by gnakm — APPROVED** [review]
Just went over all the content changes for this with Beth. LGTM!

**Pull request merged**
